test/cql-pytest/test_wasm.py

```diff
@@ -336,6 +339,7 @@ def test_f64_param(cql, test_keyspace, table1, scylla_with_wasm_only):
   (table (;0;) 1 1 funcref)
   (table (;1;) 32 externref)
   (memory (;0;) 17)
+  (export "memory" (memory 0))
```

Comment:
Why these changes?

Reply:
We need to export memory to be able to check the instance size (we're using the size of the memory as the size of the instance in the cache)

Reply:
As we've discussed on the sync, the instance may have no memory and store everything on the stack

Reply:
but why did this become important when we started to reuse instances?

Reply:
Ah, for sizing. Maybe it's better for the sizer to assume a fixed size in that case (for all the overhead).

Reply:
Do you mean a fixed size for instances that do not export memory? That could work, but it's hard to come up with a reasonable default size. Maybe it will be easier to decide after we do some testing

Reply:
ok